src/Symfony/Component/DependencyInjection/Attribute/Autoconfigure.php

```diff
@@ -20,15 +20,65 @@
 class Autoconfigure
 {
     public function __construct(
+        /**
+         * The tags to add to the service.
+         *
+         * @var array<array-key, array<array-key, mixed>>
+         */
         public ?array $tags = null,
+
+        /**
+         * The calls to be made when instantiating the service.
+         *
+         * @var array<string, array<array-key, mixed>>
+         */
         public ?array $calls = null,
+
+        /**
+         * The bindings to declare for the service.
+         *
+         * @var array<string, mixed>
+         */
         public ?array $bind = null,
+
+        /**
+         * Whether the service is lazy-loaded.
+         */
         public bool|string|null $lazy = null,
+
+        /**
+         * Whether to declare the service as public.
+         */
         public ?bool $public = null,
+
+        /**
+         * Whether to declare the service as shared.
+         */
         public ?bool $shared = null,
+
+        /**
+         * Whether to declare the service as autowired.
+         */
         public ?bool $autowire = null,
+
+        /**
+         * The properties to define when creating the service.
+         *
+         * @var array<string, mixed>
```

Comment:
```suggestion
         * @var array<string, mixed>|null
```

Reply:
Oups sorry, forgot about it! I addressed your comment :slightly_smiling_face: